pkg/defaults/defaults.go

```diff
@@ -290,7 +290,7 @@ const (
 	// ForceLocalPolicyEvalAtSource is the default value for
 	// option.ForceLocalPolicyEvalAtSource. It is enabled by default to
 	// provide backwards compatibility, it can be disabled via an option
-	ForceLocalPolicyEvalAtSource = true
+	ForceLocalPolicyEvalAtSource = false
```

Comment:
Could you elaborate further in your commit message why this is /safe/ to change the default, e.g.  will there be breakage if a user who is not explicitly setting force-local-policy-eval-at-source (and therefore relying on defaults) and the upgrade from one Cilium version to another where this is flipped?

Reply:
As per the issue description 
https://github.com/cilium/cilium/issues/15452
we should Disable the flag by default to avoid packets to be processed twice when per-endpoint routes are enabled.
When user upgrade from one Cilium version to another, user can enable via an option. When the PR get merged and it should be mentioned the same in release notes.